tests/test_astrometryModel.py

```diff
@@ -390,6 +390,12 @@ def testGetVisitTransform(self):
         self.checkGetVisitTransform(self.model1)
         self.checkGetVisitTransform(self.model2)
 
+    def test_validate(self):
+        # Model1 has 70 parameters (2 visits*20 params + (6-1) sensors*5 params)
+        # so we need 70+1 degrees of freedom to fit it.
```

Comment:
This isn't quite the right use of "degrees of freedom," I don't think; you need at least 1 degree of freedom so you need 71 data points/measurements/observations/inputs.

Reply:
Yeah, that was left over from when I was still sorting out what "degrees of freedom" meant in this context. I'm a bit disappointed that `validate` is just checking that the number is one or more, but I guess that's all there is to check. I'll reword it.

Relatedly, can we chat about whether that number should be 1 or 2 for astrometry?